pom.xml

```diff
@@ -21,13 +21,6 @@
         <tag>HEAD</tag>
     </scm>
 
-    <developers>
-        <developer>
-            <id>fbelzunc</id>
-            <name>Felix Belzunce Arcos</name>
-        </developer>
-    </developers>
-
     <properties>
         <maven.compiler.source>1.6</maven.compiler.source>
         <maven.compiler.target>1.6</maven.compiler.target>
```

Comment:
The above is a `warning`, it is not an `error`. The number of genuine errors is probably very small and quickly corrected.